cilium-cli/connectivity/builder/to_fqdns.go

```diff
@@ -20,8 +20,8 @@ func (t toFqdns) build(ct *check.ConnectivityTest, templates map[string]string)
 		WithCiliumPolicy(templates["clientEgressToFQDNsPolicyYAML"]).
 		WithFeatureRequirements(features.RequireEnabled(features.L7Proxy)).
 		WithScenarios(
-			tests.PodToWorld(tests.WithRetryDestPort(80)),
-			tests.PodToWorld2(), // resolves cilium.io.
```

Comment:
I don't understand this change - the test checks `cilium.io:443` (with the expectation that it fails, see below) and `one.one.one.one:80` - we only want to retry the later, since the former will always fail.

Reply:
:facepalm: I should have read the commit message more clearly, apoligies

Reply:
To me, it would make sense to only require retries for `PodToWorld` (the allowed one), but not `PodToWorld2` (the denied one)

Reply:
You're right, thanks for digging into this and for the explanation. I should have done that before I sent the change. Looks like the way tests are structured can make it a bit hard at times to understand what's tested vs. what's expected. I'll drop the last 2 commits (since setting retry options on `PodToWorld2` won't be necessary after all) and convert this to a pure cleanup PR. Thanks again!